llvm/test/CodeGen/DirectX/splitdouble.ll

```diff
@@ -36,6 +37,11 @@ define noundef <3 x i32> @test_vector_double_split(<3 x double> noundef %d) {
 ; CHECK-NEXT:    [[DOTUPTO116:%.*]] = insertelement <3 x i32> [[DOTUPTO015]], i32 [[DOTI1]], i64 1
 ; CHECK-NEXT:    [[TMP1:%.*]] = insertelement <3 x i32> [[DOTUPTO116]], i32 [[DOTI2]], i64 2
 ; CHECK-NEXT:    ret <3 x i32> [[TMP1]]
+;
+; DXIL-LOWER-LABEL: define noundef <3 x i32> @test_vector_double_split(
```

Comment:
Lines 40 and 41 are the same as the check-lable and check-same lines. Please consolidate similarities and only use DXIL-LOWER for differences. That means you will need to create a new check label for all the intrinsics like `@llvm.dx.splitdouble.i32`